src/pages/pupils/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/results/[attemptId].tsx

```diff
@@ -0,0 +1,11 @@
+import withFeatureFlag from "@/hocs/withFeatureFlag";
+
+const ResultsPage = () => {
+  return <div>ResultsPage</div>;
+};
+
+const FeaturedResultsPage = withFeatureFlag(
```

Comment:
As per above we don't need the feature flag here just export the ResultsPage as default.